### ISSUE TITLE
Sticky close footer on modals

### ISSUE THREAD

**Issue by McManning** [McManning/ostem#9]
Modals seem to take forever to scroll on mobile, and click-out doesn't work on iOS. Instead, add a sticky footer close button to improve accessibility.
